galleon-feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~
+  ~ Copyright 2021 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<build xmlns="urn:wildfly:feature-pack-build:3.1" producer="org.jboss.resteasy.microprofile:galleon-feature-pack">
+    <transitive>
+        <dependency group-id="org.wildfly" artifact-id="wildfly-ee-galleon-pack" translate-to-fpl="true">
```

Comment:
@jamezp Where is this `wildfly-ee-galleon-pack` defined? I tried to find in Wildfly and Galleon codebase but it seems I fail to find it.

Reply:
It looks like you have:
```
<dependency>
    <groupId>org.wildfly</groupId>
    <artifactId>wildfly-galleon-pack</artifactId>
    <version>${version.org.wildfly}</version>
    <type>zip</type>
</dependency>
```
defined in your POM. That has a transitive dependency on `org.wildfly:wildfly-ee-galleon-pack` which is likely how it's resolved. I would probably change that dependency to `org.wildfly:wildfly-ee-galleon-pack` if you want to keep it. You can leave it off since you've got a dependency on `org.jboss.resteasy:galleon-feature-pack` which also includes the dependency.

The reason the `resteasy-micoprofile` needs the `org.wildfly:wildfly-galleon-pack` is because it requires MicroProfile modules.